AnkiHubSoftware/ankihub_addon
# [BUILD-834] feat: Add split screen to the card review

## Related issues
- [BUILD-834](https://ankihub.atlassian.net/browse/BUILD-834)


## Proposed changes
This PR adds a new provisional button to trigger the split-screen functionality, which opens a dedicated WebView instance to display content from user-specified URLs.

## How to reproduce
1. Go to the ankihub and create and enable the feature flag "mh_integration"
2. On Anki click on the option -> AnkiHub -> Sync with AnkiHub
3. Go to the card review page
4. Click on the button "Toggle Split Screen"
5. This should open a webview
6. Click on one of the tabs to change the website


## Screenshots and videos

![image](https://github.com/user-attachments/assets/e2f25e44-bf83-4d84-8696-004df8288248)



[BUILD-834]: https://ankihub.atlassian.net/browse/BUILD-834?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
